src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -111,7 +111,9 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
   }, [phone, fieldName]);
 
   useEffect(() => {
-    dataSchemaRef.current![fieldName].validate = validate;
+    if (dataSchemaRef.current) {
```

Comment:
Without check it causes issues with HMR
<details>
  <summary>Error</summary>
<img width="926" alt="Screenshot 2023-11-28 at 12 26 50" src="https://github.com/okta/okta-signin-widget/assets/72614880/0941c511-aaba-43d6-85c1-530efb80fcc8">
</details>